python/lsst/qa/explorer/functors.py

```diff
@@ -365,6 +380,9 @@ class IDColumn(Column):
     _allow_difference = False
     _defaultNoDup = True
 
+    def _func(self, df):
+        return pd.Series(df.index, index=df.index)
```

Comment:
Probably a separate ticket, but I don't think a major customization point for `Functor` should be something that starts with an underscore; how to label Python methods that are effectively `protected` is always tricky, but the importance of this one swings me towards thinking it should be considered to be more `public` than `private`.

Reply:
Porting the Functors over to pipe_tasks will require a big refactoring anyway